src/js/ble_uuid_util.js

```diff
@@ -36,8 +36,23 @@
 
 module.exports.removeDashes = function(uuid) {
   if (uuid) {
-    uuid = uuid.replace(/-/g, '');
+    uuid = uuid.split('-').join('');
   }
 
   return uuid;
 };
+
+module.exports.tokenize = function(str) {
```

Comment:
@jeuxdeau What about `splitAtEvery2ndChars`? I cannot know the expected working by the name of `tokenize`.